src/Tribe/Status/Manager.php

```diff
@@ -219,4 +225,34 @@ public function return_statuses_by_action( $action, $commerce ) {
 
 	}
 
+	/**
+	 * Return an array of Statuses for a Commerce with label and stock attributes
+	 *
+	 * @since TBD
+	 *
+	 * @param $commerce string a string of the Commerce System to get statuses from
+	 *
+	 * @return array an array of statues with label and stock attributes
+	 */
+	public function return_status_options( $commerce ) {
+
+		$status_options = array();
```

Comment:
It'd be great if any of this was cached in some way so if we fetch the status options for a given commerce provider multiple times, we won't have to re-initialize the options. Perhaps make the `$status_options` variable static within this method and then index it by `$commerce`.

```php
static $status_options = array();

if ( ! empty( $status_options[ $commerce ] ) ) {
  return $status_options[ $commerce ];
}
```